code/modules/unit_tests/create_and_destroy.dm

```diff
@@ -29,6 +29,8 @@
 		/mob/dview,
 		//Template,
 		/obj/effect/mapping_helpers/custom_icon,
+		//Proccall handler to handle proccalls
+		/mob/proccall_handler,
```

Comment:
Why does this need to be blacklisted?

Reply:
it cannot be deleted

Reply:
That's only if we try and delete the original though right? any new objects the test creates should be processed just fine

Reply:
ah I see